### DISCUSSION TITLE
Example of leapp-cli --filter for ID ?

### DISCUSSION THREAD

**Discussion by nitrocode** [Noovolari/leapp#294] [Q&A]
I'm having trouble using filter

```
leapp list sessions --filter name=id
```

And I cannot find any info in the docs for an example.

**Comment by sgendrot** [comment]
You have to use the name of parameters as filter, ex;
`leapp session list --filter="Session Name=^mysession$"`
`leapp session list --filter="Named Profile=myprofile"`
`leapp profile list --filter="Profile Name=^myprofile$"`

The filter is a regex, so `myprofile` with match `xxxmyprofilexxx`. You need  to specify `^` for the beginning and `$` for the end of the string. `^myprofile$` will match exactly `myprofile` and not `xxmyprofilexx`

**Comment by andreacavagna01** [comment]
Thanks for the contribution! A more detailed answer can be prepared by @ericvilla  and @urz9999
